src/components/AudioPlayer/ProgressBar.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react'
 import { Text, View, StyleSheet } from 'react-native'
-import TrackPlayer from 'react-native-track-player'
+import TrackPlayer, { STATE_BUFFERING } from 'react-native-track-player'
 import MultiSlider from '@ptomasroos/react-native-multi-slider'
 
 // Based off of the web version, with a few major differences
 class ProgressBar extends TrackPlayer.ProgressComponent {
-  state = {
-    seeking: false,
-    seekingValue: 0,
+  constructor(props) {
```

Comment:
Is this change necessary?

Reply:
I don't think so, I just did it to follow the standard format for React class components.